### DISCUSSION TITLE
The problem with incomplete wins XCH

### DISCUSSION THREAD

**Discussion by zn1W** [Chia-Network/chia-blockchain#8182] [Support]
Hello,

Please check my Chia address:
xch1mdw5ma6vh9s927slp5cz0lrf09r5lvapsyv4rsv8fsvfsg5cmk5qyj2f3g
Solo mining. I have incomplete blocks at different heights, can you check if everything is okey? The first two awards are okey, I got a full 2x 2XCH. I have doubts about the marked heights. Can you check it ? Or send an e-mail to the appropriate support department.
From August 19, I started mining in the space. 

btw. block confirmation (737639) is in the register (wallet) but has not been added to the balance? Why is it like that?

![hist](https://user-images.githubusercontent.com/89425747/130528598-aa6eeb9a-b365-40e8-9369-8348174b02ff.png)

Regards.

**Comment by loppefaaret** [comment]
the incoming 0.25 at 06:14 looks to be a farming reward on a plot that belongs to a plotNFT, that is pooling on a pool perhaps ?
and the top incoming transaction you see, is change coming back from the prior outgoing 0.000000000001 and not a block reward

**Comment by BrandtH22** [comment]
@zn1W , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
